README.md

```diff
@@ -67,6 +67,22 @@ Make sure this file is listed in the .gitignore file, so your bot token isn't up
 For using the YouTube notifications functionality you'll need to set the `YOUTUBE TOKEN` in your `.env` file. Follow the instructions [here](https://developers.google.com/youtube/registering_an_application) to create an API key.
 
 
+# Using spreadsheet functionality
+
+* To use the spreadsheet functionality you'll need to go [here](https://console.developers.google.com/?hl=nl).  
+* Login with your google-account. Click "create project".  
+* Give the project a name and click "create". Then click on "enable apis and services".  
+* Look for "sheets" and click on "Google Sheets API". Then click on "enable".  
+* On the right-side, click "create credentials". Select "Google Sheets API". On the next question select "web server".  
+* Select "Application data" and "no, I'm not using them". Click "what credentials do I need?".  
+* Fill in the fields and for the role select "project -> editor". Select JSON if not yet selected. Click "continue".  
+* You'll automatically download the JSON. Rename it to something shorter and don't share it!  
+* Put it in the same folder as the .env-file. Go into the JSON to copy the client_email field.  
+* Add that email in google sheets like you'd add a normal user.  
```

Comment:
It was not completely clear what this meant to me. Maybe write it as "add that email in the google sheet as an editor"

Reply:
Maybe also add a sentence on what has to be done if this is implemented in the prodbot?

Reply:
Not sure if Bob would need more explanation than this.

Reply:
I would still simply add "as an editor" to the statement, just to make it clear.

Reply:
That's what the line says now